teraserver/python/modules/FlaskModule/API/user/UserQueryDevices.py

```diff
@@ -67,37 +77,49 @@ def get(self):
 
         args = parser.parse_args()
 
-        if args['uuid']:
-            args['device_uuid'] = args['uuid']
+        # as_available = args.pop('available')
+        as_projects = args.pop('projects')
+        as_enabled = args.pop('enabled')
+        as_list = args.pop('list')
+        as_with_participants = args.pop('with_participants')
+        as_with_sites = args.pop('with_sites')
+        as_with_status = args.pop('with_status')
```

Comment:
Should be "has_projects", "has_enabled", ... since otherwise it could cause confusion ("cast object as projects" instead of "object has projects").

Reply:
Changed, thank you